packages/sdk/src/dataProtector/sharing/subgraph/getProtectedDataById.ts

```diff
@@ -15,13 +14,32 @@ export async function getProtectedDataById({
         id: "${protectedDataAddress}"
       ) {
         id
+        name
         owner {
           id
         }
+        collection {
+          id
+          owner {
+            id
+          }
+        }
+        isRentable
+        isIncludedInSubscription
+        isForSale
       }
     }
   `;
-  const { protectedData }: GetProtectedDataByIdGraphQLResponse =
-    await graphQLClient.request(getProtectedDataQuery);
+  const { protectedData } = await graphQLClient.request<{
+    protectedData: {
+      id: Address;
+      name: string;
+      owner: { id: Address };
+      collection: { id: Address; owner: { id: Address } };
+      isRentable: boolean;
+      isIncludedInSubscription: boolean;
+      isForSale: boolean;
+    };
+  }>(getProtectedDataQuery);
```

Comment:
No need to extract a type and to reference it in `graphQLTypes.js` as it was only used in this file.
However we could extract this as a type and simply put it a the top of this file. Maybe cleaner?